### ISSUE TITLE
unable to import google.cloud.aiplatform.gapic.schema in version 0.3.1

### ISSUE THREAD

**Issue by khu834** [googleapis/python-aiplatform#170]
#### Environment details

  - OS type and version: Debian 10
  - Python version: 3.8.5
  - pip version: pip 20.3.3
  - `google-cloud-aiplatform` version: Version: 0.3.1

#### Steps to reproduce
I'm trying to construct a client predict request for images in a deployed object detection model endpoint. Following this example.
https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/predict_image_object_detection_sample.py

#### Code example

```python
from google.cloud.aiplatform.gapic.schema import predict
```

#### Stack trace
```
ModuleNotFoundError: No module named 'google.cloud.aiplatform.gapic.schema'
```

**Comment by dizcology** [comment]
Hi @khu834: The schema module was added only after library version 0.4.0.  The samples got merged with updated code before 0.4.0 was released to PyPI.

Please upgrade the library and it should work.

**Comment by khu834** [comment]
Thank you. That works. Closing this ticket.
